lib/PuppeteerSharp/ScreenshotOptions.cs

```diff
@@ -32,12 +42,19 @@ public class ScreenshotOptions
         /// </summary>
         /// <value>The type.</value>
         [JsonProperty("type")]
-        public string Type { get; set; }
+        public ScreenshotType? Type { get; set; }
         /// <summary>
         /// The quality of the image, between 0-100. Not applicable to png images.
         /// </summary>
         /// <value>The quality.</value>
         [JsonProperty("quality")]
         public decimal? Quality { get; set; }
+
+        internal static ScreenshotType? GetScreenshotTypeFromFile(string file)
```

Comment:
is similar to https://github.com/GoogleChrome/puppeteer/blob/master/lib/Page.js#L794?

Reply:
Yes, it matches. The AspNet.Core library has the `FileExtensionContentTypeProvider`, which resolves mime types. But I didn't want to add a dependency only to check 4 file extensions.

So that dictionary it's based on a mime type list, but it has the ScreenshotType as a result.

Reply:
do we need a test for this?

Reply:
I'd make sense

Reply:
Done @Meir017